unit_tests/server/parameters_parser.cpp

```diff
@@ -559,6 +557,33 @@ BOOST_AUTO_TEST_CASE(valid_match_urls)
     CHECK_EQUAL_RANGE(reference_2.coordinates, result_2->coordinates);
 }
 
+BOOST_AUTO_TEST_CASE(invalid_match_urls)
+{
+    std::vector<util::Coordinate> coords_1 = {{util::FloatLongitude{1}, util::FloatLatitude{2}},
+                                              {util::FloatLongitude{3}, util::FloatLatitude{4}}};
+
+    MatchParameters reference_1{};
+    reference_1.coordinates = coords_1;
+    auto result_1 = parseParameters<MatchParameters>("1,2;3,4?radiuses=unlimited;60");
+    BOOST_CHECK(result_1);
+    CHECK_EQUAL_RANGE(reference_1.timestamps, result_1->timestamps);
+    CHECK_EQUAL_RANGE(reference_1.bearings, result_1->bearings);
+    BOOST_CHECK(reference_1.radiuses != result_1->radiuses);
+    CHECK_EQUAL_RANGE(reference_1.approaches, result_1->approaches);
+    CHECK_EQUAL_RANGE(reference_1.coordinates, result_1->coordinates);
+
+    MatchParameters reference_2{};
+    reference_2.coordinates = coords_1;
+    reference_2.radiuses = {7, 60};
+    auto result_2 = parseParameters<MatchParameters>("1,2;3,4?radiuses=seventy;60");
+    BOOST_CHECK(!result_2);
+    CHECK_EQUAL_RANGE(reference_2.timestamps, result_2->timestamps);
+    CHECK_EQUAL_RANGE(reference_2.bearings, result_2->bearings);
+    BOOST_CHECK(reference_2.radiuses != result_2->radiuses);
+    CHECK_EQUAL_RANGE(reference_2.approaches, result_2->approaches);
+    CHECK_EQUAL_RANGE(reference_2.coordinates, result_2->coordinates);
+}
+
```

Comment:
What are we testing here? You didn't change the grammar or parsing, so I think this test is unnecessary.

Instead what do you think of adding ana dditional limit test here: https://github.com/Project-OSRM/osrm-backend/blob/ad7b99ec5c535ca9ec03a1ea50653e1fb8898cd8/unit_tests/library/limits.cpp#L107-L132

Reply:
@daniel-j-h ooo I was just trying to check the case of passing in a non-number. It wasn't a well thought out test-case, rather an attempt to get code coverage for [this case](https://github.com/Project-OSRM/osrm-backend/pull/4721/files/ad7b99ec5c535ca9ec03a1ea50653e1fb8898cd8#diff-89873aae5c93116cd0f09f2e1a69c320R222)

Reply:
Let me try with the limit test!!